### PR TITLE
Merge master into features

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: doc/en/cache.rst [hunk 1]
```diff
@@ -262,16 +256,96 @@ You can always peek at the content of the cache using the
...
+       'test_show_warnings.py::test_one': True,
+       'test_simple.yml::hello': True,
+       'test_smtpsimple.py::test_ehlo': True,
+       'test_step.py::TestUserHandling::test_modification': True,
+       'test_strings.py::test_valid_string[!]': True,
+       'test_tmp_path.py::test_create_file': True,
+       'test_tmpdir.py::test_create_file': True,
+       'test_tmpdir.py::test_needsfiles': True,
+       'test_unittest_db.py::MyTest::test_method1': True,
+       'test_unittest_db.py::MyTest::test_method2': True}
```
Comment (blueyed) [inline doc/en/cache.rst]: This looks unexpected?

Comment (nicoddemus) [reply doc/en/cache.rst]: Hmmm you are right, I missed that.

Probably it is picking up the cache from the previous runs; I get the same output locally even if I remove my `.pytest_cache` folder. Not sure why this is happening just now.

Regardless we can merge this and fix it on `master`, I think.

Comment (blueyed) [reply doc/en/cache.rst]: Why not fix it on master before merging?

Comment (blueyed) [reply doc/en/cache.rst]: (it is not clear to me / I don't know how that works though - I just think the less the diffs the better)

Comment (blueyed) [reply doc/en/cache.rst]: Yeah, it is also in https://github.com/pytest-dev/pytest/pull/5014/files#diff-040438ee0e8d77242351c8cc6baf8499L269 already - gonna fix it from there.

Comment (blueyed) [reply doc/en/cache.rst]: I think it should even hold the release back, since it would have wrong documentation.

Comment (blueyed) [reply doc/en/cache.rst]: But that is tagged already, so ok.

Comment (blueyed) [reply doc/en/cache.rst]: But I still think we should not merge regressions across branches.

Comment (nicoddemus) [reply doc/en/cache.rst]: No worries, I will close this for now and we can merge this later when this issue is fixed. 👍

Comment (blueyed) [reply doc/en/cache.rst]: Any idea about the issue?
Would be nice to have this merged for the version at least.. :)

Comment (nicoddemus) [reply doc/en/cache.rst]: Haven't had the time to look at it yet. 😞 

> for the version at least

What do you mean? 🤔

Comment (blueyed) [reply doc/en/cache.rst]: `--version` - to get the tag into features.  Just something minor.

**Pull request closed without merging**